index.html

```diff
@@ -0,0 +1,203 @@
+            </div>
+            <div class="submodul__image">
+                <img src="img/img4.png" alt="img4">
+            </div>
+            <div class="add__itame">
+                <div class="add__image">
+                    <img src="img/add.png" alt="add_img">
+                </div>
+                
+            </div>
+        </div>
+      </div>
+    </div>
+        <div class="footer">
+            <div class="footer__wrap">
+                <div class="footer__social">
+                    <h5 class="footer__header"><span class="colortext">Care'</span>mental</h5>
+                    <div class="footer__icon-img">
+                        <img src="img/facebook.png" alt="facebook">
+                        <img src="img/twitter.png" alt="twitter">
+                        <img src="img/inst.png" alt="inst">
+                        <img src="img/youtube.png" alt="youtube">
+                    </div>
+                    <div class="footer__social-text">We are very experienced and professional to make customers
+                        feel comfortable getting services from us</div>
+                </div>
+                <div class="footer__product">
+                    <div class="footer__product-title">Product</div>
+                    <ul class="footer__product-list">
+                        <li>Honey herbal</li>
+                        <li>Nutribee</li>
+                        <li>Honeycute</li>
+                    </ul>
+                </div>
+                <div class="footer__about">
+                    <div class="footer__about-title">About Us</div>
+                    <ul class="footer__about-list">
+                        <li>Our people</li>
+                        <li>Our activities</li>
+                        <li>Contact us</li>
+                    </ul>
+                </div>
+                <div class="footer__principles">
+                    <div class="footer__principles-title">Our principles</div>
+                    <ul class="footer__principles-list">
+                        <li>Corporate principles</li>
+                        <li>Independent inspections</li>
+                        <li>Best practices</li>
+                    </ul>
+                </div>
+                <div class="footer__info">
+                    <div class="footer__info-title">Info</div>
+                    <ul class="footer__info-list">
+                        <li>FAQ</li>
+                        <li>Cookie & help</li>
+                        <li>Privacy policy</li>
+                        <li>Terms & conditions</li>
+                        <li>Help</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+
+
+
+<script src="js/main.js"></script>
+
+</html>
```

Comment:
Принято все файлы завершать переносом строки (забавно, даже гитхаб тут на это ругнулся)
```suggestion
</html>

```